portal/code/cx-portal/src/components/styles/_fonts.scss

```diff
@@ -0,0 +1,47 @@
+@font-face {
+  font-family: FabricMDL2Icons-0;
+  src: url(https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/10.0.0/css/fabric.min.css);
```

Comment:
are we sure this is a reliable source?

Reply:
I'm not sure. I just copy/paste it from main.scss
Do we need it generally? Or can I delete it? I don't see, that we ever used it

Reply:
I don't know where this came from. Maybe when I originally copied some styles from the old portal. I think you can delete that.